packaging/windows/clisdk/bundle.wxs

```diff
@@ -45,6 +62,21 @@
   </Bundle>
 
   <Fragment>
+    <util:RegistrySearch
+       Id="VCRedist_KeyExists"
+       Variable="VCRedist_KeyExists"
+       Root="HKLM"
+       Key="SOFTWARE\Microsoft\VisualStudio\14.0\VC\Runtimes\$(var.Platform)"
+       Value="Version"
+       Result="exists" />
+    <util:RegistrySearch
+       Id="VCRedist_KeyValue"
+       Variable="VCRedist_KeyValue"
+       Root="HKLM"
+       Key="SOFTWARE\Microsoft\VisualStudio\14.0\VC\Runtimes\$(var.Platform)"
+       Value="Version"
+       Result="value" />
```

Comment:
Good catch.  We always want to search the 32-bit registry (even the 64 bit CRT writes there) so I will need to be explicit here.